docs/how-to-guides/develop-a-flow/develop-chat-flow.md

```diff
@@ -0,0 +1,69 @@
+# Develop Chat flow
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../faq.md#stable-vs-experimental).
+:::
+
+From this document, you can learn how to develop a chat flow by writing a flow yaml from scratch. You can 
+find additional information about flow yaml schema in [Flow YAML Schema](../../reference/flow-yaml-schema-reference.md).
+
+## Flow input data
+
+The most important elements that differentiate a chat flow from a standard flow are **Chat Input** and **Chat History**. A chat flow can have multiple inputs, but **Chat History** and **Chat Input** are required inputs in chat flow.
```

Comment:
you have already bolded the text, so that you don't need to CAPITALIZE them.

Reply:
same as below, only natural line start needs capitalize.

Reply:
Changed to lowercase